### ISSUE TITLE
examples for the pedagogy section

### ISSUE THREAD

**Issue by giladgressel** [udacity/mentor_encyclopedia#2]
Hey all,

I think that the pedagogy section would really benefit from concrete examples.  It's one thing to say something, it's much better if we can illuminate the pedagogical concepts with concrete examples

I'm opening this an issue because while I could find examples in the machine learning forums, I don't think they would lend themselves well to this purpose, I fear they would have too many buzzwords or be too specific to the nature of machine learning to understand how the mentor is pushing the student.

I believe that we would be most benefited by examples that are easily understood by our entire mentor team, so I'm wondering what ND would suit that best.

Introduction to Programming came to mind, because I think most ND's have a programming aspect to them, and require some elements of this?  

Are there any intro programming mentors that would be willing to dig around for some good examples of pedagogical posts that we could use as examples?

@KalindiFonda tagging you because I think you work in that ND?

**Comment by giladgressel** [comment]
hey I just saw that the debugging section actually really overalps with the pedagogy section, and has good examples

I think this kind of "here is the principle" followed by "here is the example" is what I was thinking of.

**Comment by Mrk-Nguyen** [comment]
I agree, examples posts can help. Intro to Programming would be a good example. We're coming out with Predictive Analytics for Business which is non-code focus so I can pull example posts from there when we launch in the next few weeks.
